src/services/OrderService.ts

```diff
@@ -470,6 +470,52 @@ const fetchShippingAddress = async (contactMechId: string): Promise<any> => {
   return shippingAddress;
 }
 
+const getCustomerContactNumber = async (orderId: string): Promise<any> => {
+  let contactNumber = ''
+  try {
+    let resp: any = await api({
+      url: "performFind",
+      method: "get",
+      params: {
+        "entityName": "OrderContactMech",
+        "inputFields": {
+          orderId,
+          "contactMechPurposeTypeId": "PHONE_SHIPPING"
+        },
+        "fieldList": ["orderId", "contactMechPurposeTypeId", "contactMechId"],
+        "viewSize": 1
+      }
+    })
+
+    if (!hasError(resp)) {
+      const contactMechId = resp.data.docs[0].contactMechId
+      resp = await api({
+        url: "performFind",
+        method: "get",
+        params: {
+          "entityName": "TelecomNumber",
+          "inputFields": {
+            contactMechId,
+          },
+          "fieldList": ["contactNumber", "contactMechId"],
```

Comment:
Fetch country code, area code as well.  So in fact you can remove select fields here and fetch all the fields from the TelecomNumber table..

Reply:
Made changes as per the discussion.